src/main/scala/engine/SimulationHistory.scala

```diff
@@ -12,11 +13,19 @@ object SimulationHistory{
 
   var history:History = List()
 
-  /**Initiliaze the [[History]] of this simulation
+  var mutations: Option[List[Mutation]] = None
+
+  /** Initialize the [[History]] of this simulation
    * @param environment the initial environment of the first [[Generation]] */
   def initialize(environment: Environment): Unit =
     history = Generation(environment, generateInitialCouple) :: history
 
+  /** Introduce a new mutation */
+  def introduceMutation(mutation: Mutation): Unit = mutations match {
+    case None => mutations = Some(List(mutation))
+    case _ => mutations = Some(mutations.get.::(mutation))
```

Comment:
quando vuoi usare l'operatore `::` il nuovo elemento lo devi mettere in testa: `mutation :: mutations`